src/platform/CMakeLists.txt

```diff
@@ -29,6 +27,9 @@ include_directories(platform
 target_link_libraries(platform
   update)
 
+target_link_libraries(platform
```

Comment:
Why not put this in the `target_link_libraries` above such that it looks like this:
```
target_link_libraries(platform
  update
  utils)
```

Reply:
OK, see reply to Gerry

Reply:
lol, didn't realize @gerboland was reviewing as well.  I'll just resolve this one.